gatsby-config.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  pathPrefix: process.env.PATH_PREFIX || "/",
```

Comment:
please name this something like `KEPLOY_PATH_PREFIX` because I got confused with `PREFIX_PATHS` gatsby inbuilt variable